operator/src/test/java/oracle/kubernetes/operator/PodWatcherTest.java

```diff
@@ -359,6 +360,7 @@ void whenPodDeletedOnFirstRead_runNextStep() {
       testSupport.runSteps(watcher.waitForDelete(createPod(), terminalStep));
 
       assertThat(terminalStep.wasRun(), is(true));
```

Comment:
with your new test that the step was executed exactly once, you no longer need the test for `wasRun()`

Reply:
I have removed the redundant check in [0ea2a3d](https://github.com/oracle/weblogic-kubernetes-operator/pull/3412/commits/0ea2a3dc5c26b618115b7db257fa810a9d7a24b6). Thanks.